clusters/nerc-ocp-test/griot-grits/application-set.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: griot-and-grits-apps
+spec:
+  goTemplate: true
+  goTemplateOptions: ["missingkey=error"]
+  syncPolicy:
+    preserveResourcesOnDeletion: false
+    applicationsSync: "sync"
```

Comment:
@dystewart Just FYI this is causing sync errors due to `applicationsSync` not existing in the spec. Confirmed with `oc explain applicationset.spec.syncPolicy` on ocp-infra cluster:
```
error validating data: [ValidationError(ApplicationSet.spec): unknown field "goTemplate" in io.argoproj.v1alpha1.ApplicationSet.spec, ValidationError(ApplicationSet.spec): unknown field "goTemplateOptions" in io.argoproj.v1alpha1.ApplicationSet.spec, ValidationError(ApplicationSet.spec.syncPolicy): unknown field "applicationsSync" in io.argoproj.v1alpha1.ApplicationSet.spec.syncPolicy]
```